docs/specification/current.md

```diff
@@ -226,11 +226,43 @@ It is the CDS Service's responsibility to check prefetched data against its temp
 
 #### Prefetch tokens
 
-A prefetch token is a placeholder in a prefetch template that is replaced by a value from the hook's context to construct the FHIR URL used to request the prefetch data.
+A prefetch token is a placeholder in a prefetch template that is *_replaced by information from the hook's context_* to construct the FHIR URL used to request the prefetch data.
 
-Prefetch tokens MUST be delimited by `{{` and `}}`, and MUST contain only the qualified path to a hook context field.
+Prefetch tokens MUST be delimited by `{{` and `}}`, and MUST contain only the qualified path to a hook context field *_or one of the following user identifiers: `userPractitionerId`, 'userPractitioneRolerId', `userPatientId`, or `userRelatedPersonId`_*.
+
+Individual hooks specify which of their `context` fields can be used as prefetch tokens. Only root-level fields with a primitive value within the `context` object are eligible to be used as prefetch tokens. For example, `{{context.medication.id}}` is not a valid prefetch token because it attempts to access the `id` field of the `medication` field.
+
+##### Prefetch tokens identifying the user
+A prefetch template enables a CDS Service to learn more about the current user through a FHIR read, like so:
+```
+{
+  "prefetch": {
+    "user": "{{context.userId}}"
+  }
+}
+```
+or though a FHIR search:
+```
+{
+  "prefetch": {
+    "user": "PractitionerRole?_id={{userPractitionerId}}&_include=PractitionerRole:practitioner"
+  }
+}
+```
+
+A prefetch template may include any of the following prefetch tokens:
+
+
+Token | Description
+---|---
+`{{userPractitionerId}}` | FHIR id of the Practitioner resource corresponding to the current user. 
+`{{userPractitionerRoleId}}`|FHIR id of the Practitioner resource corresponding to the current user. 
+`{{userPatientId}}`|FHIR id of the Practitioner resource corresponding to the current user. 
+`{{userRelatedPersonId}}`|FHIR id of the Practitioner resource corresponding to the current user. 
```

Comment:
Update "Practitioner" to be the appropriate type
```suggestion
`{{userPractitionerRoleId}}`|FHIR id of the PractitionerRole resource corresponding to the current user. 
`{{userPatientId}}`|FHIR id of the Patient resource corresponding to the current user. 
`{{userRelatedPersonId}}`|FHIR id of the RelatedPerson resource corresponding to the current user. 
```

Reply:
thank you, Dennis!